Platform/ElkhartlakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -805,8 +804,11 @@ TccModePostMemConfig (
   BIOS_SETTINGS                             *PolicyConfig;
   TCC_STREAM_CONFIGURATION                  *StreamConfig;
   TCC_CFG_DATA                              *TccCfgData;
+  FEATURES_CFG_DATA                         *FeaturesCfgData;
 
-  TccCfgData = (TCC_CFG_DATA *) FindConfigDataByTag(CDATA_TCC_TAG);
+  TccCfgData      = (TCC_CFG_DATA *) FindConfigDataByTag (CDATA_TCC_TAG);
+  FeaturesCfgData = (FEATURES_CFG_DATA *) FindConfigDataByTag (CDATA_FEATURES_TAG);
+  ASSERT (FeaturesCfgData != NULL);

```

Comment:
Do we really need ASSERT? ASSERT is always enabled regardless of DEBUG/RELEASE in SBL. Can we use default value to continue to boot if FeaturesCfgData is not found?

Reply:
done